assets/sass/modules/content-blocks/_hero.scss

```diff
@@ -25,7 +25,7 @@
 		justify-content: center;
 		margin: 0 auto;
 		max-width: 50%;
-		min-height: rem(500); // height of hero.
+		min-height: rem(700); // height of hero.
```

Comment:
Wondering if we should use a smaller min-height for mobile?  700px takes up a lot of real estate and could present the background image/video at a weird crop.  Maybe half it to 350px?

Reply:
Fixed, they are the same now.